roles/allocations/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+allocations_config_path: /etc/allocations
+allocations_docker_network: allocations
+allocations_docker_network_subnet: 172.18.11.0/24
+# Corresponds to the default user 1001 used in s2i builds
+allocations_user: 1001
+
+allocations_balance_service_docker_image: docker.chameleoncloud.org/allocations/balance-service:latest
+allocaitons_balance_service_name: allocations-balance-service
+allocations_balance_service_port: 8915
+
+redis_host: chi.uc.chameleoncloud.org
+# comma sperated auth users
+allocations_auth_users: 97d6f370dbc5451da5521633eeda5ad6
```

Comment:
Might be easier to have this be a YAML array, and then use the `join` filter when rendering it out on the template. In general I think it's easier to keep lists as lists in YAML

```jinja
comma_separated_list = "{{ true_list | join(",") }}"
```